jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibExtension.java

```diff
@@ -221,6 +227,22 @@ public String getContainerizingMode() {
     return property != null ? property : containerizingMode.get();
   }
 
+  /**
+   * Returns the configurationName property while setting it to the value of the system property if
+   * present.
+   *
+   * @return The configurationName property
+   */
+  @Input
+  @Optional
+  public Property<String> getConfigurationName() {
+    String property = System.getProperty(PropertyNames.CONFIGURATION_NAME);
+    if (property != null) {
```

Comment:
FTR, this can be problematic and should be fixed. https://github.com/GoogleContainerTools/jib/pull/3034#issuecomment-792844429